docs/deploy.md

```diff
@@ -76,3 +75,60 @@ Finally, add a `Procfile` to your project with the following:
 web: node bin/rendertron
 ```
 
+# Deploying Rendertron in a docker container
+
+Based on Puppeteer instructions we can create a docker image that bundles a headless chrome and rendertron. We can start from node 14 base image.
+
+For more information about chrome installation please see the pupeteer page: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker
+
+Before creating the docker file you must create the config.json used by rendertron.
+
+```
+{
+    "cache": "filesystem",
```

Comment:
Is there a specific reason for using the filesystem cache? Could people opt for, say, the memory cache, too?

Reply:
Yes they can. This was just a random example. But to be more clear I will change this paragraph.

But on docker I believe the most appropriate cache system is google datastore. Both the filesystem and memory will require to define the cacheMaxEntries property. Or else we can end with a container that it is using a lot of resources with just cache data.
Have you considered to use memcache for example? I believe it would be well suited when running rendertron in a docker environment.